src/mame/wyse/wy50.cpp

```diff
@@ -36,6 +36,8 @@
 #include "wy50kb.h"
 #include "video/scn2674.h"
 #include "screen.h"
+#include "speaker.h"
+#include "sound/beep.h"
```

Comment:
These should be kept organised – see here: https://docs.mamedev.org/contributing/cxx.html#structural-organization